### PR TITLE
Add missing icons to menus

### PR DESCRIPTION
**Pull request by buckmelanoma**
This patch adds several missing icons to the following menu items:

mainwindow.cpp:
- invert selection
- set location
- rename
- servers
- server options
- server stats

torrentfilesview.cpp:
- download
- not download
- rename

Sample Screenshots:

Before:
before

After:
after

### CODE REVIEW

**Comment by equeim** [comment]
Please add new icons to the list in data/CMakeLists.txt (https://github.com/equeim/tremotesf2/blob/master/data/CMakeLists.txt#L96) so that they will be automatically deployed in Windows builds (only if they exist in breeze icon theme, you can check it by searching [here](https://github.com/KDE/breeze-icons/find/v5.89.0)).

**Comment by buckmelanoma** [comment]
I added the new icon file names to data/CMakeLists.txt where needed.  Build is clean here on Linux.

I noticed there were some existing duplicate entries in there (ex. "view-refresh.svg"). Is this needed when there are duplicate calls in the source?

**Comment by equeim** [comment]
No, duplicates are an error (it shouldn't break anything though, it will just make generated regex for finding icons bigger and slower). You can remove them if you want.

Could you also sqash commits into one and rebase is on top of current master (I merged some changes that make GitHub Actions run for Windows faster)?

File: data/CMakeLists.txt [hunk 1]
```diff
@@ -98,14 +98,18 @@ if (WIN32)
             application-exit.svg
             applications-utilities.svg
             configure.svg
+	    dialog-cancel.svg
```
Comment (equeim) [inline data/CMakeLists.txt:101]: There seems to be an issue with indentation :)

**Review by equeim — CHANGES_REQUESTED** [review]

**Comment by equeim** [comment]
Nice, thank you!

**Pull request merged**